include/pclomp/ndt_omp.h

```diff
@@ -58,6 +58,16 @@ namespace pclomp
 		DIRECT1
 	};
 
+	struct NdtResult
+	{
+		Eigen::Matrix4f pose;
```

Comment:
According to [this document](http://eigen.tuxfamily.org/dox/group__TopicStructHavingEigenMembers.html) we need to declare `EIGEN_MAKE_ALIGNED_OPERATOR_NEW` within this struct.

Reply:
Resolved: https://github.com/tier4/ndt_omp/pull/16/commits/763211bddcfb6e328c81d517a104826a2fb3fc91